connection.js

```diff
@@ -1351,12 +1373,12 @@ class Connection {
             this.errors++;
             return this.respond(503, "Use MAIL before RCPT");
         }
-
+        // Track relay state
+        this.relay_warn = this.relaying;
```

Comment:
Don't you have to do this check in the _respond functions?

Reply:
No - I'm making sure this.relay_warn is set to whatever this.relaying is set to *before* the plugins are run as I'm checking to see if they changed the value of this.relaying during the hooks.

Reply:
I guess I should have called it this.relay_saved ...